### DISCUSSION TITLE
test – Test Giscus

### DISCUSSION THREAD

**Discussion by giscus** [r-leyshon/quarto-giscus#2] [General]
# test – Test Giscus

http://localhost:4736/

**Comment by r-leyshon** [comment]
foobar

**Comment by r-leyshon** [comment]
test 2

**Comment by r-leyshon** [comment]
# Testing

~~markdown~~ **functionality**

[some link](https://www.google.com.mx/)

![some image](https://i.imgur.com/MiCmAtN.jpeg)

**Comment by r-leyshon** [comment]
updating quarto config did not result in a loss of comments
